worlds/AutoWorld.py

```diff
@@ -348,6 +349,9 @@ class World(metaclass=AutoWorldRegister):
     world_version: ClassVar[Version] = Version(0, 0, 0)
     """Optional world version loaded from archipelago.json"""
 
+    rule_caching_enabled: ClassVar[bool] = False
+    """Enable or disable the rule result caching system"""
```

Comment:
With the newest change (CachedRuleBuilderWorld), I'm *slightly* concerned about this, but I haven't checked the code yet. What happens when you just turn this one for a normal world?

Reply:
I didn't think about this now that we've split off the cached world as its own thing. This might need to be replaced by `isinstance` instead of an attribute.

Reply:
Removed the prop from `World` and the docs. Still keeping it on `CachedRuleBuilderWorld` since rule instantiation needs to know and doing an `isinstance` would put us back to circular import hell.

If someone puts this prop onto their world without using the cached world, it's their own fault